docs/produto/acceptance-test.md

```diff
@@ -0,0 +1,70 @@
+| Não conseguiu realizar |          |          |          |          |          |          |
+| Sim, com dificuldade   |          |          |          |          |          |          |
+| Sim, com facilidade    |          |          |          |          |          |          |      
+| Tempo de execução      |          |          |          |          |          |          |         
+
+Ao final da avaliação os testadores serão submetidos às seguintes perguntas(que também deverão ser registradas e preenchidas em cada relatório):      
+1. Como você avaliaria sua experiência ao utilizar o aplicativo? (de 1 a 5, como 1 sendo uma experiência muito ruim e 5 sendo uma experiência muito boa)       
+2. Você utilizaria esse app no seu dia-a-dia? Por que?         
+3. Como você avalia a identidade visual do aplicativo?        
+4. Acha que os ícones são representativos?        
+5. Tem alguma reclamação sobre alguma tela específica?     
+6. Sentiu falta de alguma instrução/informação?
+7. As funcionalidades apresentadas no aplicativo corresponderam as suas expectativas em relação a proposta do aplicativo?      
+
+# Roteiro 
+## Instrução Inicial
+Todos os testes deverão ser iniciados com a exposição do objetivo do produto, qual o contexto do desenvolvimento e qual o objetivo do teste que será realizado, enfatizando que o produto será avaliado e não a capacidade do usuário de realizar as tarefas. Além disso, é necessário:        
+* Como se dará o andamento do teste;       
+* Solicitar que o usuário expresse sempre o que está pensando e o que está tentando fazer;      
+* Relembrar o usuário de que o produto está em processo de aprimoramento, então não deve se preocupar em criticar pois sua crítica será de extrema importância para o resultado final.       
+
+## Realização de Tarefas
+* Tarefa 1: Será solicitado que o usuário realize seu cadastro na aplicação.      
+* Tarefa 2: Será solicitado que o usuário Cadastre uma ocorrência.      
+* Tarefa 3: Será solicitado que o usuário Visualize uma ocorrência.     
+* Tarefa 4: Será solicitado que o usuário visualize estatística do crime de roubo a veículo.
+* Tarefa 5: Será solicitado que o usuário visualize estatísticas de uma cidade.
+* Tarefa 6: Será solicitado que o usuário avalie um bairro.         
+
+## Critérios de Sucesso   
+* O sucesso de cada tarefa é o usuário conseguir executá-la. 
+* Para cada uma das tarefas foi determinado um tempo mínimo, então caso usuário consiga executá-la dentro do tempo, demonstrará um bom padrão da usabilidade e reforçará o resultado positivo. 
+* Feedbacks dos testadores serem positivos(em sua maioria) também demonstrarão o sucesso dos testes.   
+
+### Heurísticas de Nielsen
+Conforme descrito no [documento de qualidade](https://fga-eps-mds.github.io/2020.1-stay-safe-docs/produto/quality/) do Stay Safe, os testes avaliarão se a aplicação atende as Heurísticas de Nielsen. Caso atenda, demonstra então que aplicação possui boa navegabilidade e usabilidade.
```

Comment:
You can omit the word "aplicação" or add an article "a" in "demonstra então que **aplicação** possui...". (line 62)